### DISCUSSION TITLE
Anycubic Kobra 2 MAX config needed

### DISCUSSION THREAD

**Discussion by tadejales** [OrcaSlicer/OrcaSlicer#2461] [Q&A]
I just received my new 3d printer and I cant find suitable parameters that would yield good prints.

I am using 0.6mm nozzle.

This is the config that I am curently working on. 
[Anycubic Kobra 2 Max - PLA.zip](https://github.com/SoftFever/OrcaSlicer/files/13042134/Anycubic.Kobra.2.Max.-.PLA.zip)

**Comment by djodjo67** [comment]
you have profile orcaslicer for nozzle 0.4

**Reply by tadejales** [reply]
But it is marlin based, not klipper as far as I understand.

**Reply by docdawning** [reply]
Kobra 2 Max is for sure a Klipper machine. I believe the Kobra Max is Marlin, but I'm not sure.

**Comment by Heildnix** [comment]
> I just received my new 3d printer and I cant find suitable parameters that would yield good prints.
> 
> I am using 0.6mm nozzle.
> 
> This is the config that I am curently working on. [Anycubic Kobra 2 Max - PLA.zip](https://github.com/SoftFever/OrcaSlicer/files/13042134/Anycubic.Kobra.2.Max.-.PLA.zip)

Have you worked any further on this profiel?
Do you also have an profile for the standard 0.4 nozzle?

**Reply by tadejales** [reply]
Niether of them is working properly.

**Comment by Parchbrot** [comment]
I created a profile for Orca Slicer copying manually from Prusa Slicer and put it into Orca Slicer (changed the available Kobra 2 settings).
[K2M_Orca_Slicer_Profile.zip](https://github.com/SoftFever/OrcaSlicer/files/13855427/K2M_Orca_Slicer_Profile.zip)
I also added the stl file for the bed.
Maybe something can be improved, but it is working with v1.8.1

**Comment by Hiabu1** [comment]
Hi, thank you a lot.

I got a big Problem, when i Sliced the file with Orca and upload it with the anycubic cloud, it tells me this file is broken.

I really don't know what it would be.
Maybe somebody can help?

**Comment by larados2** [comment]
hello. Does Orcaslicer show you the tumbnails on the Kobra 2 screen? I think I have it configured correctly but it doesn't show me the thumbnails or the remaining time or anything.
tumbs = 110 x 230 png

**Reply by Parchbrot** [reply]
hi,
thumbnail isnt showing yet. try to fix this...

**Comment by larados2** [comment]
[k2p-orcaslicer profile.zip](https://github.com/SoftFever/OrcaSlicer/files/14068578/k2p-orcaslicer.profile.zip)
These are the profiles that I use and they work well for me at 0.2mm
